DEMO.md

```diff
@@ -35,6 +35,10 @@
   - The conveyor will begin moving and parts will appear at random intervals.
   - This step will also bring up the individual nodes for tool planning, trajectory planning, 
     perception and robot execution, each in its own terminal window (don't close the windows).
+  - The connection between the linear carriage that carries the arm and the rail that it travels along can 
```

Comment:
This whole paragraph should probably go in section 2 of the "Application bringup" as it pertains to the rail joint motion problem.